src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing.mdx

```diff
@@ -19,23 +19,23 @@ For how to view your user count in the UI, see [Billing-related UI](/docs/accoun
+* **Our primary New Relic One pricing version.** This version is also known as "New Relic One with core users." It was released January 12, 2022. Organizations on this version have access to a third user type: the [core user](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type). And this version has different [billing calculations](#user-count) and [user downgrade rules](#user-downgrade-rules). 
+* **The "New Relic One without core users" version.** If your New Relic organization existed prior to January 12, 2022, and hasn't switched to the newer version, this is your version. Organizations on this version can add only basic users and full platform users; they don't have access to core users. This pricing version will be increasingly deprecated over time. See the [rules for this version](#non-core).
 
 Tips for determining which version you're on: 
 * If your organization has core users or has the ability to add core users, you're on our primary version. If you can only add basic users and full platform users, you're on the "New Relic One without core users" version. You can view your users by going to the [**User management** UI](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks/#where). 
 * Another way to determine your version is by going to the [**Manage your plan** UI](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/billing-ui). If you see "New Relic One - Users", that means you're on the "New Relic One without core users" version.  
 
 For more about the differences between these versions, see [Core users release](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/core-users-release).
 
-## Billable user calculation [#user-count]
-
-The main way to understand your billable user count is to go to the [usage UI](/docs/accounts/accounts-billing/general-account-settings/introduction-account-settings/#pricing). If you need more detail than the UI provides, you can also run [usage-related queries](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts). 
+## Billable user calculation for primary pricing version (with core users) [#user-count]
 
 <Callout variant="important">
-These billing rules apply for organizations on our [primary New Relic One pricing version](#pricing-versions) (the version with core users). 
+These rules apply for organizations on our [primary New Relic One pricing version](#pricing-versions) (the version with core users). 
 </Callout>
 
+The main way to understand your billable user count is to go to the [usage UI](/docs/accounts/accounts-billing/general-account-settings/introduction-account-settings/#pricing). If you need more detail than the UI provides, you can also run [usage-related queries](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts). 
```

Comment:
How about something like...?

```suggestion
You can use the [usage UI](/docs/accounts/accounts-billing/general-account-settings/introduction-account-settings/#pricing) to get an overview of your billable user count. If you need more detail than the UI provides, you can also run [usage-related queries](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts) in New Relic One. 
```

Maybe it isn't obvious where to run this usage-related query?